tethysext/atcore/templates/atcore/resource_workflows/report_workflow_results_view.html

```diff
@@ -0,0 +1,234 @@
+    }
+
+    .legend-list-item {
+        margin-top: 5px;
+        margin-bottom: 5px;
+    }
+
+    .legend-list-item .flatmark {
+        display: inline-block;
+        max-width: 310px;
+        overflow: hidden;
+    }
+
+    .legend-item {
+        position: relative;
+    }
+
+    .legend-item p {
+        padding-left: 60px;
+        padding-top: 2px;
+    }
+
+    .legend-item .color-box {
+        width: 40px;
+        height: 10px;
+        display: inline-block;
+        position: absolute;
+        left: 5px;
+        top: 5px;
+    }
+    .modebar{
+      display: none !important;
+    }
+    [contentEditable=true]:empty:not(:focus)::before{
+        content:attr(data-ph)
+    }
+    .add-border {
+      margin-top: 10px;
+      margin-bottom: 40px;
+      padding: 5px;
+      border-radius: 5px;
+      border: 1px solid gray;
+    }
+
+    @media print {
+      .map-view-result {
+        width:700px; height:600px;
+      }
+      div:empty {
+          display: none;
+      }
+      div {
+        border-style: none !important;
+      }
+      body {
+        color-adjust: exact;
+        -webkit-print-color-adjust: exact;
+        transform: scale(1);
+      }
+      div[id*="resultType-"] {
+        padding-top: 20px;
+        page-break-inside: avoid;
+      }
+      .add-border {
+        margin-top: 0;
+        margin-bottom: 0;
+      }
+      #inner-app-content {
+        overflow: visible;
+      }
+      #app-content {
+        overflow: visible;
+      }
+      #btnPrint, #app-header, #app-actions,  #app-actions > *, .results-tabs {
+        display: none;
+      }
+      #app-content-wrapper.show-nav #app-content {
+        padding-right: 0;
+        transform: translate3d(0,0,0);
+        overflow: true;
+      }
+      #app-navigation {
+        display: none;
+      }
+      #app-content-wrapper #app-content {
+        padding-top: 0;
+        overflow: visible;
+      }
+      #app-content-wrapper {
+        overflow-x: visible;
+      }
+      a[href]:after {
+        content: none !important;
+      }
+    }
+  </style>
+{% endblock %}
+
+{% block import_gizmos %}
+  {{ block.super }}
+  {% import_gizmo_dependency plotly_view %}
+  {% import_gizmo_dependency bokeh_view %}
+  {% import_gizmo_dependency datatable_view %}
+  {% import_gizmo_dependency map_view %}
+{% endblock %}
+
+{% block app_content %}
+  {{ block.super }}
+
+  <form id="report_form">
+  <div id="report_workflow_data">
+    {% if has_tabular_data %}
+      <div class="tab-data">
+        {% for tab_data_title, tab_data in tabular_data.items %}
+        <h2>{{ tab_data_title }}</h2>
+        {% for k, v in tab_data.items %}
+        <p><b>{{ k|title }}:</b> {{ v }}</p>
+        {% endfor %}
+        {% endfor %}
+      </div>
+    {% endif %}
+    <div contentEditable=true class="add-border" data-ph="General Note"></div>
+    <!-- Try to find a way to move the script (one line below) into the component. -->
+    <script>
+      var maps = [];
```

Comment:
What are these variables used for?

Reply:
these are used to loop over results when there is more than one map, table ,etc.